src/store/keys/OpportunityContextKeys.ts

```diff
@@ -163,5 +163,27 @@ export enum OpportunityContextKeys {
     CUSTOM_FIELD_98 = 'opp.csf98',
     CUSTOM_FIELD_99 = 'opp.csf99',
 
-    CUSTOM_FIELD_100 = 'opp.csf100'
+    CUSTOM_FIELD_100 = 'opp.csf100',
```

Comment:
Why do you think opportunity has more the 100 custom fields?

According to API it has exactly 100?
https://api.outreach.io/api/v2/docs#opportunity

Reply:
This custom fields exceeding for Account and Opportunity was mentioned last week on the All-hands-meeting, Martin Petrla have worked on that.